src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -17,17 +17,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Data.SqlClient" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.IO.Packaging" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.ServiceModel.Duplex" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.ServiceModel.Http" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.ServiceModel.NetTcp" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.ServiceModel.Security" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.Threading.AccessControl" Version="4.4.0-preview1-25302-01" />
-    <PackageReference Include="System.Private.ServiceModel" Version="4.4.0-preview1-25302-01" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.4.0-preview3-25423-02" />
```

Comment:
Yes, it is expected. `Microsoft.NetCore.App 2.0.0-preview3-25426-01` references to packages of the build `25423-02` (built in 6/23). However, for `ServiceModel` related packages, they only have `25423-01` built on that day, so `25423-01` is used for `ServiceModel` related packages.